package.json

```diff
@@ -7,8 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint -f stylish",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "jest --passWithNoTests",
```

Comment:
It looks like you're ignoring the test results here? Is that really what you want?

The renovate team recommend you have _working_ tests before allowing renovate to automerge dependencies.

Reply:
[https://github.com/Taiki0807/next.js-template/actions/runs/4745704781](https://github.com/Taiki0807/next.js-template/actions/runs/4745704781)
I am ignoring this due to a failed Github Actions run.

Reply:
a2e4f8f88cd514ead4c2f8166a277a78f194d777
I have a sample component ready to test, and the Github Actions come through.